spec/routes/directory.spec.js

```diff
@@ -58,13 +58,15 @@ describe('directory route', function () {
         .end(function(err, res) {
           expect(res.statusCode).to.equal(200);
           var $ = cheerio.load(res.text);
-          expect($('message').text()).to.equal(
-`We found multiple people, reply with:
-1 for Thor Girl
-2 for Frog Thor
-3 for Thor
-Or start over`
-          );
+          var responseMessage = $('message').text()
+          var lines = responseMessage.split('\n');
+
+          var pattern = /^\d (for Thor Girl|for Frog Thor|for Thor)$/;
```

Comment:
I have a few concerns about the changes you introduced to this test.

1. For me it is hard to understand, some regexes, lines (more concepts to deal with)
1. It introduces multiple expectations even though conceptually all of them are part of the same expectation. Usually this is not a good practice.

This might be a flaky test. Just for the text part you have a lot of possibilities, you can think on permutations. We'd like to narrow down the scope of the test just to make an assertion for the "thor" search criteria. Which for this example will always return:

```
1 for thor Girl
2 for Frog Thor # and so on. (You can see this as an immutable property of the universe.)
```
If something changes in the search algorithm, I'd like to have a safety net that warn me about those changes, even if it is something trivial as the order of results.

I'd love to hear your thoughts on this.